build/rule.go

```diff
@@ -265,11 +265,14 @@ func (r *Rule) SetAttr(key string, val Expr) {
 // If the rule has no such attribute or the attribute is not an identifier or number,
 // AttrLiteral returns "".
 func (r *Rule) AttrLiteral(key string) string {
-	lit, ok := r.Attr(key).(*Ident)
-	if !ok {
-		return ""
+	value := r.Attr(key)
+	if ident, ok := value.(*Ident); ok {
+		return ident.Name
 	}
-	return lit.Name
+	if literal, ok := value.(*LiteralExpr); ok {
```

Comment:
`A LiteralExpr represents a literal number.`
I'm not sure if it makes sense to have a function that returns either an identifier, or a number stored as string.

Reply:
There are usages that expect the value to be either `1` or `True`, for instance.